babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -788,15 +788,71 @@ SqlIndex SqlCreateTableIndex(Span s) :
 SqlNode SqlExecMacro() :
 {
     SqlIdentifier macro;
+    SqlNodeList paramNames = new SqlNodeList(getPos());
```

Comment:
Prefer using a class object to encapsulate both name and value.

Reply:
Since this is more of a design decision, let's discuss it in the design doc.